test/src/jmh/kotlin/performance/BenchmarkTestBase.kt

```diff
@@ -0,0 +1,31 @@
+package performance
+
+import graphql.nadel.Nadel
+import graphql.nadel.NadelExecutionInput
+import graphql.nadel.tests.next.NadelIntegrationTest
+import org.openjdk.jmh.annotations.Fork
+import org.openjdk.jmh.annotations.Measurement
+import org.openjdk.jmh.annotations.Scope
+import org.openjdk.jmh.annotations.State
+import org.openjdk.jmh.annotations.Warmup
+
+@State(Scope.Benchmark)
+@Warmup(iterations = 5, time = 5)
+@Measurement(iterations = 5)
+@Fork(2)
```

Comment:
Not a blocker for this PR: For GraphQL Java benchmarks we had slightly fewer warmup and measurement runs.

Usually 2 for warmup and 2-3 for measurement

Because the GraphQL Java benchmark tests run on every commit we had to be more careful there about the total running time for the tests (we arbitrarily set a limit of about 1 hour).